packages/curve-ui-kit/src/shared/ui/SymbolCell.tsx

```diff
@@ -0,0 +1,28 @@
+import { Stack, Typography } from '@mui/material'
+import { t } from '@ui-kit/lib/i18n'
+import { MetricSize } from '@ui-kit/shared/ui/Metric'
+import { TokenIcon } from '@ui-kit/shared/ui/TokenIcon'
+import { WithSkeleton } from '@ui-kit/shared/ui/WithSkeleton'
+
+type SymbolCellProps = {
+  label: string
+  symbol: string | undefined | null
+  tokenAddress: string | undefined | null
+  loading: boolean
+  size?: keyof Pick<typeof MetricSize, 'small' | 'medium'>
+}
+
+/** Mimics the style of Metric but is used for cells that only have a symbol and a token icon. */
+export const SymbolCell = ({ label, symbol, tokenAddress, loading, size = 'medium' }: SymbolCellProps) => (
```

Comment:
Part of me wonders if we could just make the type of the `value` prop in `Metric` to also be able to be a `ReactNode`, then you can put the label with tokenicon in it. But this works for now I suppose